troncli/version.py

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
put version tag in a separate file is a bit of overkill

Reply:
A version file for this is created.
The standard provides "__version__.py"